Tiltfile

```
@@ -14,16 +16,21 @@ docker_build_with_restart(
     '.',
     only=[
         './bin',
+        'localhost.pem',
+        'localhost-key.pem',
     ],
     dockerfile="dev.dockerfile",
-    entrypoint='/app/build/gitops ui run -l',
+    entrypoint='/app/build/gitops-server --tls-cert-file=build/localhost.pem --tls-private-key-file=build/localhost-key.pem',
```

```suggestion
    entrypoint='/app/build/gitops-server -l --tls-cert-file=build/localhost.pem --tls-private-key-file=build/localhost-key.pem',
```

`-l` logging flag vanished somewhere.

Also if we wanted to swap the tls flags for an `--insecure` I would not be averse.

---

Added `-l` back. I will respectfully ignore the `--insecure` suggestion right now, since I think this PR has enough changes as it is.